src/renderers/dom/shared/devtools/ReactDOMUnknownPropertyDevtool.js

```diff
@@ -22,6 +22,15 @@ if (__DEV__) {
     dangerouslySetInnerHTML: true,
     key: true,
     ref: true,
+
+    defaultValue: true,
+    valueLink: true,
+    defaultChecked: true,
+    checkedLink: true,
+    innerHTML: true,
+    suppressContentEditableWarning: true,
+    onFocusIn: true,
+    onFocusOut: true,
```

Comment:
`innerHTML`, `onFocusIn`, and `onFocusOut` already emit warnings telling the user not to use them.  Having it also emit this warning is just confusing, so I blacklisted those tags that we already warn about.